### PR TITLE
updated UserRolesServiceIntegrationTest

### LINKED ISSUE #61: [Testing] UserRole Service Test
### Resources: 
Resource from Azhya: 
[Spring Boot Testing Basics: How to Unit Test & Integration Test REST Controllers](https://www.youtube.com/watch?v=pNiRNRgi5Ws)

Additional Resources:
[Test Driven Development with Spring Boot - Sannidhi Jalukar, Madhura Bhave](https://www.youtube.com/watch?v=s9vt6UJiHg4)

-----
### Methods: 
As a small reminder, we only need the minimum amount of controller code needed to actually run the test (Calvin's example should show that). 
Note: It's expected to fail. 
Action item: Please go ahead and set up the minimum amount of code (in the Controller layer) for your assigned testing table that you will need to create your test. 

Your tests should have the following methods: 
- getAll
- getById
- Create
- Update
- Delete

### PR DESCRIPTION
**Pull request by pyaeger**
Will add reviewers once built and ready for review as per Calvin's note in Discord.

### CODE REVIEW

**Comment by pyaeger** [comment]
Built this as far as my current understanding allows.

**Review by FullSol — COMMENTED** [review]
This looks goo, just missing a couple of methods. We can go over this and complete it this afternoon.

**Comment by pyaeger** [comment]
Reworked UserRolesService Integration Test based on Calvin's ImgPostService Integration Test.

**Review by FullSol — APPROVED** [review]

File: src/main/java/com/amplifier/services/UserRolesServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.amplifier.services;
+
+public class UserRolesServiceImpl implements UserRolesService {
+
+	@Override
+	public boolean isValidUserRoleViolated(Integer id, String role) {
```
Comment (L3viathon) [inline src/main/java/com/amplifier/services/UserRolesServiceImpl.java]: I'm a bit confused by this method. I'm not sure what this method is trying to do?

**Review by L3viathon — CHANGES_REQUESTED** [review]
Outside of the method in the UserRolesServiceImpl, nice! Gonna set it to Request changes just to be on the safe side....

**Pull request merged**